src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -365,6 +365,13 @@ public void OnClientProcessExit(string stdError)
             this.clientExitErrorMessage = stdError;
             this.clientExited.Set();
 
+            if (!this.IsOperationComplete() && stdError != string.Empty)
```

Comment:
Should the logging be consolidated at one place..? Also use string.isnullorwhitespace instead of !=

---
In reply to: [214585772](https://github.com/Microsoft/vstest/pull/1752#discussion_r214585772) [](ancestors = 214585772)